:scripts/hw4_solution Mtweedietest.r

```diff
@@ -0,0 +1,139 @@
+# Assignment 1:  
+library(tweedie) 
+library(ggplot2)
+
+simTweedieTest <-  
+  function(N){ 
+    t.test( 
+      rtweedie(N, mu=10000, phi=100, power=1.9), 
+      mu=10000 
+    )$p.value 
+  } 
+
+
+# Assignment 2:  
+
+maxcores <- 8
+Cores <- min(parallel::detectCores(), maxcores)
```

Comment:
Also, this approach is correct for task 2 but not task 3. Look into it further when you have more time available :)